src/coreclr/pal/src/exception/seh-unwind.cpp

```diff
@@ -54,7 +54,7 @@ Module Name:
 
 #endif // HOST_UNIX
 
-#if defined(TARGET_OSX) && defined(TARGET_ARM64)
+#if defined(TARGET_OSX) && defined(HOST_ARM64)
```

Comment:
Nit: Also fix `#endif // defined(TARGET_OSX) && defined(TARGET_ARM64)`

Reply:
Done, thanks for spotting it.